backend/routes/download.js

```diff
@@ -240,6 +240,8 @@ router.route('/csv/:showId')
                       studentEmail: `${entryData.studentUsername}@rit.edu`,
                       studentFirstName: user ? user.firstName : null,
                       studentLastName: user ? user.lastName : null,
+                      studentHometown: user ? user.hometown : null,
+                      studentDisplayName: user ? user.displayName : null,
```

Comment:
The models already contain hometown and displayname, we just need to tell the software to grab that data for the CSV